addons/stock_transport/models/stock_picking_batch.py

```diff
@@ -0,0 +1,112 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models, api
+
+
+class StockPickingBatch(models.Model):
+    _inherit = 'stock.picking.batch'
+
+    @api.model
+    def _default_end_date(self):
+        self.end_date = fields.datetime.now()
+
+    vehicle_id = fields.Many2one('fleet.vehicle', string="Vehicle")
+    vehicle_category_id = fields.Many2one(
+        'fleet.vehicle.model.category', string="Vehicle Category", compute='_compute_vehicle_category_id', inverse='_inverse_vehicle_category_id', store=True)
+    dock_id = fields.Many2one('stock.location', string="Dock Location", domain="[('is_a_dock', '=', True)]")
+    vehicle_details = fields.Char(
+        "Vehicle Details", compute='_compute_vehicle_details')
+    max_weight = fields.Float(string="Max Weight (Kg)",
+                              related='vehicle_category_id.max_weight')
+    max_volume = fields.Float(string="Max Volume (m³)",
+                              related='vehicle_category_id.max_volume')
+    driver_id = fields.Many2one(
+        related="vehicle_id.driver_id", string="Driver")
+    used_weight_percentage = fields.Float(
+        string="Weight %", compute='_compute_used_weight_percentage', store=True)
+    used_volume_percentage = fields.Float(
+        string="Volume %", compute='_compute_used_volume_percentage', store=True)
+    max_picking_weight = fields.Float(string="Weight", compute='_compute_used_weight_percentage', store=True)
+    max_picking_volume = fields.Float(string="Volume", compute='_compute_used_volume_percentage', store=True)
+    end_date = fields.Datetime('End Date', default=_default_end_date)
+    picking_info = fields.Char(string="Picking Type Information",
```

Comment:
default=fields.Datetime.now

Reply:
I have tried withdefault=fields.Datetime.now but for some reason `end date` button;s string shows 'Add start date' as shown in a, so to handle this I have tried with the method and by this, it displays the correct string, why is it like that I’m not sure.